models/public/colorization-v2/colorization-v2.md

```diff
@@ -0,0 +1,81 @@
+# colorization-v2
+
+## Use Case and High-Level Description
+
+The `colorization-v2` model is one of the [colorization](https://arxiv.org/pdf/1603.08511)
+group of models designed to perform image colorization. For details
+about this family of models, check out the [repository](https://github.com/richzhang/colorization).
+
+The gray-scale or BGR image is first converted to LAB color space. 
```

Comment:
I suggest rephrasing to `The gray-scale or BGR image should be converted to LAB color space first.` The same goes for other descriptions.

Reply:
@Wovchena corrected